crates/rpc/Cargo.toml

```
@@ -10,3 +10,14 @@ rust-version.workspace = true
 version.workspace = true
 
 [dependencies]
+alloy-primitives.workspace = true
+anyhow.workspace = true
+discv5.workspace = true
+futures.workspace = true
+serde.workspace = true
+serde_yaml.workspace = true
+serde_json.workspace = true
+tokio.workspace = true
+warp.workspace=true
+clap.workspace=true
+thiserror.workspace=true
```

```suggestion
warp.workspace = true
thiserror.workspace = true
```

clap isn't used in this crate, but I am sure there are others